### ISSUE TITLE
vcxproj parsing fails if pch configuration is mixed

### ISSUE THREAD

**Issue by spaun2002** [Caphyon/clang-power-tools#145]
```
VERBOSE: PCH cpp name: stdafx.cpp
VERBOSE: PCH header name: 
Get-ProjectStdafxDir : Cannot bind argument to parameter 'pchHeaderName' because it is an empty string.
At C:\Users\***\AppData\Local\Microsoft\VisualStudio\15.0_45d4eb56\Extensions\gc5s3pti.xjv\clang-build.ps1:1594 
char:54
+                                       -pchHeaderName $stdafxHeader
+                                       -pchHeaderName $stdafxHeader
    + CategoryInfo          : InvalidData: (:) [Get-ProjectStdafxDir], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Get-ProjectStdafxDir
VERBOSE: PCH not enabled for this project!
Did not expect that
At C:\Users\nikonyuc\AppData\Local\Microsoft\VisualStudio\15.0_45d4eb56\Extensions\gc5s3pti.xjv\clang-build.ps1:1039 
char:9
+         throw "Did not expect that"
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Did not expect that:String) [], RuntimeException
    + FullyQualifiedErrorId : Did not expect that
```
The project has
```
    <ClCompile Include="stdafx.cpp">
      <PrecompiledHeader>Create</PrecompiledHeader>
    </ClCompile>
```
but the precompiled header file is not specified. Instead `<ForceInclude> stdafx.h` is used.

**Comment by hero101111** [comment]
Hello,

We have identified the issue here, and an updated version of Clang Power Tools will be released today.

Thank you for your feedback

**Comment by cpp-red-lion** [comment]
This issue was fixed in version `2.2` (just published)
Thank you for your feedback !
